app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestCatAndDogYears:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, human_age",
+        [
+            pytest.param(0, 0, [0, 0]),
+            pytest.param(14, 14, [0, 0]),
+            pytest.param(15, 15, [1, 1]),
+            pytest.param(23, 23, [1, 1]),
+            pytest.param(24, 24, [2, 2]),
+            pytest.param(27, 27, [2, 2]),
+            pytest.param(28, 28, [3, 2]),
+            pytest.param(100, 100, [21, 17]),
+        ]
+    )
```

Comment:
Add cases with negative numbers and incorrect type of data

Reply:
Please ensure to include testing for error handling as well

Reply:
Could you please ensure implementation of the item specified in the checklist?

`if the function receives an incorrect type of data it raises the correct exception.`